chassis/core/src/main/java/com/griddynamics/jagger/util/generators/ConfigurationGenerator.java

```diff
@@ -0,0 +1,99 @@
+package com.griddynamics.jagger.util.generators;
+
+import com.griddynamics.jagger.engine.e1.aggregator.session.BasicAggregator;
+import com.griddynamics.jagger.engine.e1.aggregator.workload.MetricLogProcessor;
+import com.griddynamics.jagger.engine.e1.aggregator.workload.ProfilerLogProcessor;
+import com.griddynamics.jagger.engine.e1.aggregator.workload.WorkloadAggregator;
+import com.griddynamics.jagger.engine.e1.collector.BasicSessionCollector;
+import com.griddynamics.jagger.engine.e1.collector.MasterWorkloadCollector;
+import com.griddynamics.jagger.master.DistributionListener;
+import com.griddynamics.jagger.master.configuration.Configuration;
+import com.griddynamics.jagger.master.configuration.SessionExecutionListener;
+import com.griddynamics.jagger.master.configuration.Task;
+import com.griddynamics.jagger.user.test.configurations.JTestSuite;
+import org.springframework.beans.factory.support.ManagedList;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Generates {@link Configuration} entity
+ * from {@link JTestSuite} object.
+ */
+public class ConfigurationGenerator {
+
+    private BasicSessionCollector basicSessionCollector;
+    private MasterWorkloadCollector e1MasterCollector;
+    private BasicAggregator basicAggregator;
+    private WorkloadAggregator e1ScenarioAggregator;
+    private MetricLogProcessor metricLogProcessor;
+    private ProfilerLogProcessor profilerLogProcessor;
+    private List<JTestSuite> userConfigurations;
+
+    public void setUserConfigurations(List<JTestSuite> userConfigurations) {
+        this.userConfigurations = userConfigurations;
+    }
+
+    public Configuration generate() {
+        return generate(userConfigurations.iterator().next());
+    }
+
+    /**
+     * Generates {@link Configuration} from {@code jTestConfiguration}.
+     *
+     * @param jTestConfiguration user configuration.
+     * @return jagger configuration.
+     */
+    public Configuration generate(JTestSuite jTestConfiguration) {
```

Comment:
For me there is some naming mismatch - is this param a suite or it is a **configuration** of suite? May be class naming **JTestSuite** is not correct? If not - as I see, this param name is **testSuite**